tests/components/switchbee/test_config_flow.py

```diff
@@ -0,0 +1,202 @@
+"""Test the SwitchBee Smart Home config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries
+from homeassistant.components.switchbee.config_flow import DeviceType, SwitchBeeError
+from homeassistant.components.switchbee.const import CONF_SWITCHES_AS_LIGHTS, DOMAIN
+from homeassistant.const import CONF_DEVICES, CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import RESULT_TYPE_FORM, FlowResultType
+
+from tests.common import MockConfigEntry
+from tests.components.switchbee import (
```

Comment:
```suggestion
from . import (
```
Use relative imports